bee-storage/bee-storage-rocksdb/src/access/fetch.rs

```diff
@@ -52,3 +54,25 @@ impl Fetch<Hash, MilestoneIndex> for Storage {
         }
     }
 }
+
+#[async_trait::async_trait]
+impl Fetch<MessageId, Message> for Storage {
+    type Error = OpError;
+    async fn fetch(&self, message_id: &MessageId) -> Result<Option<Message>, Self::Error>
+    where
+        Self: Sized,
+    {
+        let message_id_to_message = self.inner.cf_handle(MESSAGE_ID_TO_MESSAGE).unwrap();
+
+        let mut message_id_buf: Vec<u8> = Vec::new();
+        message_id.pack(&mut message_id_buf).unwrap();
```

Comment:
`MessageId` is `AsRef<[u8]>` so you shouldn't have ti pack to a vector

Reply:
Was it not working ? I saw you resolved but it's still there

Reply:
I forgot to push the changes D: